### PR TITLE
Add test case for concat operation

### PR DESCRIPTION
**Pull request by prigoyal**
for repro, we should support comparison operators

### CODE REVIEW

**Comment by abadams** [comment]
See #16

**Comment by prigoyal** [comment]
oops, I pushed again to disable the test, need stamp again @abadams

**Comment by prigoyal** [comment]
merging, CI is botched due to resources allocation

File: test/test_execution_engine.cc [hunk 1]
```diff
@@ -45,6 +45,24 @@ struct ATenCompilationUnitTest : public ::testing::Test {
   }
 };
 
+TEST_F(ATenCompilationUnitTest, DISABLED_Concat) {
```
Comment (nicolasvasilache) [inline test/test_execution_engine.cc]: This is dead code, I see this has been activated in master, still in the future for such cases, please make a test that does something and:
1.  expect the throw / failure that is expected
2. open a proper issue
3. tag the issue in the test as TODO(#number)

Thanks!

**Pull request merged**